package.json

```diff
@@ -42,30 +49,25 @@
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-import": "2.26.0",
     "express": "4.17.3",
-    "fastify-swagger": "5.1.1",
+    "fastify-static": "4.6.1",
```

Comment:
Could you please use `@fastify/static@5.0.0` instead of?. fastify-static has been deprecated. fastify/fastify#3733